packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -19,10 +20,19 @@ export function convertRuntimeToPlugin(
   return async function build({ workPath }: { workPath: string }) {
     const opts = { cwd: workPath };
     const files = await glob('**', opts);
+
     delete files['vercel.json']; // Builders/Runtimes didn't have vercel.json
-    const entrypoints = await glob(`api/**/*${ext}`, opts);
+
+    const entrypointPattern = `api/**/*${ext}`;
+    const entrypoints = await glob(entrypointPattern, opts);
     const pages: { [key: string]: any } = {};
-    const traceDir = join(workPath, '.output', 'runtime-traced-files');
```

Comment:
Thanks to this change, we're no longer using a single directory for all CLI Plugins.

Instead, just like with `vercel-plugin-node`, one per CLI Plugin is used. I don't see an issue with that.